src/components/connections/ConnectionSidePanel/ConnectionSidePanel.tsx

```diff
@@ -49,10 +51,16 @@ const ConnectionSidePanel = () => {
         return;
       }
 
-      await api.connectionsUpdate(generateConnectionData(connectionData, activeTile?.id), {
+      const response = await api.connectionsUpdate(generateConnectionData(connectionData, activeTile?.id), {
         headers: basicHeaders(),
       });
-      shouldRenderConnections(true);
+      const { connectionId }: NewConnection = response.data || {};
+      const gtfsStopId = connectionData.find(s => s.stopType === StopType.GTFS)?.id;
+      const osmStopId = connectionData.find(s => s.stopType === StopType.OSM)?.id;
+      setImportedConnections(oldConnections => [
+        ...oldConnections,
+        { id: connectionId, exported: false, gtfsStopId, osmStopId },
```

Comment:
Ok, tutaj potrzebowałeś tego identyfikatora connectiona. Wszystko jasne.

Reply:
Przeglądałem requesty, to on chyba nie jest używany, ale niech już tak zostanie.

Reply:
Ok, zmerguję jak ten PR: https://github.com/technologiesforaccessibility/osmintegrator-backend/pull/121 będzie zaakceptowany